Trimble.Modus.Components/Enums/ButtonStyle.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Trimble.Modus.Components.Enums
+{
+    public enum ButtonStyle
+    {
+        None,
```

Comment:
Here too, `None` shouldn't be here. By default, it should be `Filled` Can you also rename `Fill` with `Filled`?